test_autoarray/mask/derive/test_zoom_2d.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+
+import autoarray as aa
+
+
+def test__quantities():

```

Comment:
This is a perfect usecase for parametrize. Even if not using parametrize it would be better as multiple separate tests